### PR TITLE
Joss edits

### PR DESCRIPTION
**Pull request by aylapear**
- Fixing DOI's
- Fixing reference formatting

### CODE REVIEW

File: paper/paper.bib [hunk 1]
```diff
@@ -214,7 +215,7 @@ @article{laskin2020
 }
 
 @book{mcelreath2016,
-	title = {Statistical rethinking: a Bayesian course with examples in R and Stan},
+	title = {Statistical rethinking: a {B}ayesian course with examples in {R} and {S}tan},
```
Comment (nehill197) [inline paper/paper.bib:218]: I'd be inclined to capitalize "Rethinking" too.

**Review by nehill197 — CHANGES_REQUESTED** [review]
One minor suggestion on another capitalization for "Statistical **R**ethinking"

File: paper/paper.bib [hunk 2]
```diff
@@ -12,7 +12,7 @@ @article{fuller2007
 }
 
 @article{wittmer2005,
-	title = {Population dynamics of the endangered mountain ecotype of woodland caribou (Rangifer tarandus caribou) in British Columbia, Canada},
+	title = {Population dynamics of the endangered mountain ecotype of woodland caribou (Rangifer tarandus caribou) in {B}ritish {C}olumbia, {C}anada},
```
Comment (nehill197) [inline paper/paper.bib:15]:
```suggestion
	title = {Population dynamics of the endangered mountain ecotype of woodland caribou ({R}angifer tarandus caribou) in {B}ritish {C}olumbia, {C}anada},
```

**Comment by aylapear** [comment]
@editorialbot check repository

**Pull request merged**